content/PGP/Card_edit.adoc

```diff
@@ -8,7 +8,10 @@ link:https://developers.yubico.com/ykneo-openpgp/SecurityAdvisory%202015-04-14.h
 
 == Introduction
 
-To configure the OpenPGP application on a YubiKey you can use GnuPG.
+To configure the OpenPGP application on a YubiKey you can use link:https://www.gnupg.org[GnuPG].
+
+It would be worthwhile to peruse the link:https://www.gnupg.org/documentation/[GnuPG documentation],
```

Comment:
I think we can be more clear on our expectation here, something like:

"The documentation in this section assumes the reader has a general understanding of how GnuPG works. For more in-depth documentation see the [GnuPG documentation], especially the parts ... may be useful."